collectd/files/plugin/collectd_zookeeper.py

```diff
@@ -25,6 +25,8 @@
 import collectd
 import socket
 
+# Default sampling interval
+INTERVAL = 50
```

Comment:
That's not the case for all the plugins. Only for those that are heavy-loading the monitored service.